### PR TITLE
Fix issue #19

### PR DESCRIPTION
**Pull request by giiba**
Adds haptic feedback to space key.

### CODE REVIEW

File: qml/keys/SpaceKey.qml [hunk 1]
```diff
@@ -56,6 +56,8 @@ ActionKey {
         }
 
         onPressed: {
+            if (maliit_input_method.useHapticFeedback)
```
Comment (dobey) [inline qml/keys/SpaceKey.qml]: Please use open and closing braces around the block in if statements, to keep consistent with the rest of the file, like so:
```
if (foo) {
    do_something();
}
```

**Review by dobey — CHANGES_REQUESTED** [review]

File: qml/keys/SpaceKey.qml (hunk 1 above)
Comment (dobey) [reply qml/keys/SpaceKey.qml]: Also, how does this behave when doing long press? The vibration keeps happening until the key is released?

Comment (giiba) [reply qml/keys/SpaceKey.qml]: I will clean that up when I get a chance this weekend. Thabks for the pointers.

As for behaviour, no, it vibrates just once when pressed and works as expected in touchpad mode. I really just copy+pasted the line from another key's file, as the lack of feedback was annoying me; it shouldn't cause any trouble.

**Comment by giiba** [comment]
Didn't realize Github made it so easy to make changes. That should fix the syntax.

**Comment by Flohack74** [comment]
How can adding braces for a single line instruction actually fix smth? I dont get it...

**Comment by giiba** [comment]
It wasn't broken, the braces are for clarity I assume.

File: qml/keys/SpaceKey.qml [hunk 2]
```diff
@@ -56,6 +56,9 @@ ActionKey {
         }
 
         onPressed: {
+            if (maliit_input_method.useHapticFeedback) {
+                pressEffect.start();
+               }
```
Comment (dobey) [inline qml/keys/SpaceKey.qml]: Please fix the alignment of the closing brace. It should be aligned with the `i` in the `if` statement (4 spaces indented from the parent `onPressed: {` line).

**Review by dobey — CHANGES_REQUESTED** [review]

**Comment by dobey** [comment]
> How can adding braces for a single line instruction actually fix smth? I dont get it...

It's not a fix. It's keeping the code consistent, and ensure clarity of separation in the statements. It makes it easier to avoid accidental errors, and save having to add braces in the future if another statement needs to be added inside the if block.

**Comment by giiba** [comment]
I think I have it this time.

**Comment by Flohack74** [comment]
> It's not a fix. It's keeping the code consistent, and ensure clarity of separation in the statements. It makes it easier to avoid accidental errors, and save having to add braces in the future if another statement needs to be added inside the if block.

Yeah thats true, but then this PR does not fix issue #19:
image
So  it needs another title. If we merge and close this one, issue #19 will also be closed, and I think thats wrong.

**Comment by Flohack74** [comment]
> It's not a fix. It's keeping the code consistent, and ensure clarity of separation in the statements. It makes it easier to avoid accidental errors, and save having to add braces in the future if another statement needs to be added inside the if block.

Sorry I am stupid, I looked only at a subset of the changes.

**Review by dobey — APPROVED** [review]

**Comment by kugiigi** [comment]
Should we also add audio feedback?
All other keys with `overridePressArea `set to true has both haptic and audio feedback.
Or should I create a new PR since this is already merged?

EDIT: Okay so Merged = Closed. New PR then 😄

**Pull request merged**